src/main/java/com/blackducksoftware/integration/hub/api/component/ComponentRestService.java

```diff
@@ -48,4 +50,20 @@ public ComponentRestService(final RestConnection restConnection, final Gson gson
         return allComponents;
     }
 
+    public ComponentItem getExactComponentMatch(String id, String groupId, String artifactId, String version)
```

Comment:
I think perhaps there is a mis-understanding. The id parameter is used here, as far as i can tell:

`final String idSegment = StringUtils.join(new String[] { "id", id }, ':');`

and in your sample, that will be 'maven'. 

Here, 'maven' is not an id, it is the forge type.